dashboard/app/views/levels/_dialogHelper.js

```diff
@@ -35,14 +35,24 @@
     // Use our prefabricated dialog content.
     $(".modal-body").append($("#" + type + "-dialogcontent").clone(true));
 
-    // Clicking the okay button in the dialog box dismisses it.
+    // Clicking the okay button in the dialog box dismisses it, and calls the callback.
     $(".modal-body #ok-button").click(function () {
       dialog.hide();
+      callback && callback();
+    });
+
+    // Clicking the cancel button in the dialog box dismisses it.
+    $(".modal-body #cancel-button").click(function () {
+      dialog.hide();
     });
 
     dialog.show();
   }
 
+  window.showStartOverDialog = function(callback) {
```

Comment:
@sfilman do you have a vision for resolving the issue which Laurel mentioned (red button attracts rather than deters clicks)?